drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableEditorMenusTest.java

```diff
@@ -338,6 +343,8 @@ public void setup() {
 
         final GuidedDecisionTableEditorPresenter wrapped = new GuidedDecisionTableEditorPresenter(view,
                                                                                                   dtServiceCaller,
+                                                                                                  docks,
```

Comment:
Either use `@Mock` or `mock(..)` unless you had good reason?!

Reply:
Fixed, the test done for the base covers this unused mock.